src/components/TextInput/TextInputLabel/index.js

```diff
@@ -1,31 +1,32 @@
-import React, {PureComponent} from 'react';
+import React, {useRef, useEffect} from 'react';
 import {Animated} from 'react-native';
 import styles from '../../../styles/styles';
 import {propTypes, defaultProps} from './TextInputLabelPropTypes';
 import CONST from '../../../CONST';
 
-class TextInputLabel extends PureComponent {
```

Comment:
There's no noticeable performance improvement but we can use `memo` which is equivalent to `PureComponent`